src/modules/musig/tests_impl.h

```diff
@@ -1139,26 +1135,26 @@ void musig_test_vectors_signverify(void) {
         }
 
         expected = c->error != MUSIG_AGGNONCE;
-        CHECK(expected == secp256k1_musig_aggnonce_parse(ctx, &aggnonce, vector->aggnonces[c->aggnonce_index]));
+        CHECK(expected == secp256k1_musig_aggnonce_parse(CTX, &aggnonce, vector->aggnonces[c->aggnonce_index]));
         if (!expected) {
             continue;
         }
-        CHECK(secp256k1_musig_nonce_process(ctx, &session, &aggnonce, vector->msgs[c->msg_index], &keyagg_cache, NULL));
+        CHECK(secp256k1_musig_nonce_process(CTX, &session, &aggnonce, vector->msgs[c->msg_index], &keyagg_cache, NULL));
 
-        CHECK(secp256k1_ec_pubkey_parse(ctx, &pubkey, vector->pubkeys[0], sizeof(vector->pubkeys[0])));
+        CHECK(secp256k1_ec_pubkey_parse(CTX, &pubkey, vector->pubkeys[0], sizeof(vector->pubkeys[0])));
         musig_test_set_secnonce(&secnonce, vector->secnonces[c->secnonce_index], &pubkey);
         {
             /* In the last test vector we sign with an invalid secnonce, which
              * triggers an illegal_callback. Hence, we need to use a custom
              * context that does not abort in this case. */
-            secp256k1_context *ctx_tmp = secp256k1_context_create(SECP256K1_CONTEXT_NONE);
+            secp256k1_context *CTX_tmp = secp256k1_context_create(SECP256K1_CONTEXT_NONE);
             int32_t ecount = 0;
-            secp256k1_context_set_error_callback(ctx_tmp, counting_illegal_callback_fn, &ecount);
-            secp256k1_context_set_illegal_callback(ctx_tmp, counting_illegal_callback_fn, &ecount);
+            secp256k1_context_set_error_callback(CTX_tmp, counting_illegal_callback_fn, &ecount);
+            secp256k1_context_set_illegal_callback(CTX_tmp, counting_illegal_callback_fn, &ecount);
             expected = c->error != MUSIG_SECNONCE;
-            CHECK(expected == secp256k1_musig_partial_sign(ctx_tmp, &partial_sig, &secnonce, &keypair, &keyagg_cache, &session));
+            CHECK(expected == secp256k1_musig_partial_sign(CTX_tmp, &partial_sig, &secnonce, &keypair, &keyagg_cache, &session));
             CHECK((!expected) == ecount);
-            secp256k1_context_destroy(ctx_tmp);
+            secp256k1_context_destroy(CTX_tmp);
```

Comment:
this should remain `ctx_tmp` in lower case

Reply:
good catch, fixed